Content.Client/Clothing/ClientClothingSystem.cs

```diff
@@ -342,12 +342,13 @@ private void RenderEquipment(EntityUid equipee, EntityUid equipment, string slot
 
             if (displacementData is not null)
             {
+                // Frontier: revise race check
                 //Checking that the state is not tied to the current race. In this case we don't need to use the displacement maps.
-                if (layerData.State is not null && inventory.SpeciesId is not null && layerData.State.EndsWith(inventory.SpeciesId))
+                //if (layerData.State is not null && inventory.SpeciesId is not null && layerData.State.EndsWith(inventory.SpeciesId))
+                //    continue;
+                if (layer.State.Name is not null && inventory.SpeciesId is not null && layer.State.Name.EndsWith(inventory.SpeciesId))
                     continue;
-
-                if (_displacement.TryAddDisplacement(displacementData, sprite, index, key, revealedLayers))
-                    index++;
+                // End Frontier: revise race check
```

Comment:
```suggestion
                // End Frontier: revise race check

                if (_displacement.TryAddDisplacement(displacementData, sprite, index, key, revealedLayers))
                    index++;
```
Still need to add displacement for layers that don't have them.  The snippet was only intended to change the layers that should've been skipped.